src/main/java/org/opensearch/knn/indices/ModelDao.java

```diff
@@ -199,8 +201,10 @@ public void create(ActionListener<CreateIndexResponse> actionListener) throws IO
             if (isCreated()) {
                 return;
             }
-
-            CreateIndexRequest request = new CreateIndexRequest(MODEL_INDEX_NAME).mapping("_doc", getMapping(), XContentType.JSON)
+            String mapping = Strings.toString(
+                JsonXContent.contentBuilder().startObject().startObject(MapperService.SINGLE_MAPPING_NAME).endObject().endObject()
```

Comment:
getMapping contains the logic to load our model-index mapping. Why did this get deleted?

Reply:
Ack, will investigate and restore the original logic